api/Models/Location/Locations.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Scv.Api.Models.Location;
+
+public class Locations
```

Comment:
It might be better to implement as an `IList<Location>`; `Location: IList<Location>`.  Then you can treat any instance of it as an actual list, which would feel more natural.